man/systemd.network.xml

```diff
@@ -331,6 +331,13 @@
             defaults to <literal>ipv6</literal>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>LinkLocalAddressingFallback=</varname></term>
+          <listitem>
+            <para>Takes a boolean. If set to <literal>yes</literal>, enables link-local address autoconfiguration
+            when DHCP fails. Defaults to <literal>no</literal>.</para>
```

Comment:
Maybe, takes `ipv4` or `no`. Currently, this supports only ipv4 but in the future it may also support ipv6.

Reply:
Just updated that support yes no and ipv4